python/API_Examples_Python.py

```diff
@@ -0,0 +1,200 @@
+import json     # python3 -m pip install simplejson
+import requests # python3 -m pip install requests
+import pprint   # python3 -m pip install pprintpp
+import example_sign_order
+
+
+"""
+The following is a Python Script showing the use of current API's
+The following information must be inputted before using
+user_information 
+    - traderAddr 
+    - email
+    - password
+example_sign_order.py
+- post_data
+    - traderAddr
+- Private Key
+
+"""
+testnet_url = "https://testnet271828.dex.top"
+mainnet_url = "https://dex.top"
+user_information = {"token": "",
+                    "traderAddr": "",
+                    "email": "",
+                    "password": ""}
+
+def get_market_info():
+    """
+    Get relevant market information such as market contract address and token codes for signature.
+    """
+    resp = requests.get(mainnet_url + "/v1/market")
+    if resp.status_code == requests.codes.ok:
+        pprint.pprint(resp.json())
+    else:
+        print("ERROR: Get Market Information request did not work. ")
```

Comment:
How about `print("Failed to get market information:", resp.json())`?
Ditto for the rest.

Reply:
changed the format for all error cases